example/storage/eventstore_test.go

```diff
@@ -54,47 +62,63 @@ func TestPostcard_Repositories(t *testing.T) {
 		{
 			name: "postgres_simple",
 			repository: func() eventstore.EventStore[*postcard.Postcard] {
-				repo, err := storage.NewDefaultSimplePostcardRepository(context.Background(), db)
+				repo, err := storage.NewDefaultSimplePostcardRepository(context.Background(), postgresDB)
 				require.NoError(t, err)
 				return repo
 			}(),
 		},
 		{
 			name: "postgres_simple_custom",
 			repository: func() eventstore.EventStore[*postcard.Postcard] {
-				repo, err := storage.NewCustomSimplePostcardRepository(context.Background(), db)
+				repo, err := storage.NewCustomSimplePostcardRepository(context.Background(), postgresDB)
 				require.NoError(t, err)
 				return repo
 			}(),
 		},
 		{
 			name: "postgres_simple_anonymized",
 			repository: func() eventstore.EventStore[*postcard.Postcard] {
-				repo, err := storage.NewSimpleAnonymizingPostcardRepository(context.Background(), db)
+				repo, err := storage.NewSimpleAnonymizingPostcardRepository(context.Background(), postgresDB)
 				require.NoError(t, err)
 				return repo
 			}(),
 		},
 		{
 			name: "postgres_mapping",
 			repository: func() eventstore.EventStore[*postcard.Postcard] {
-				repo, err := storage.NewDefaultMappingPostgresRepository(context.Background(), db)
+				repo, err := storage.NewDefaultMappingPostgresRepository(context.Background(), postgresDB)
 				require.NoError(t, err)
 				return repo
 			}(),
 		},
 		{
-			name: "postgres_mapping",
+			name: "postgres_mapping_custom",
 			repository: func() eventstore.EventStore[*postcard.Postcard] {
-				repo, err := storage.NewCustomMappingPostcardRepository(context.Background(), db)
+				repo, err := storage.NewCustomMappingPostcardRepository(context.Background(), postgresDB)
```

Comment:
wow, lots of constructors
maybe `mapping` and `anonymizing` could be `bool`s in a config?

Reply:
or worse, options/builders 🧟

Reply:
I guess there is still some work to be done with those constructors and those ones are primarily done for tests mainly in order to explain different scenarios. Is that right, @m110?

Reply:
Ah I see, those are just constructors in the example.
The lib already has a config :o